src/widgets/mobile-view/ui/mobile-nav-item/mobile-nav-item.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames/bind';
+import Image, { StaticImageData } from 'next/image';
+
+import { DropdownArrow } from '@/shared/icons/dropdown-arrow';
+
+import styles from './mobile-nav-item.module.scss';
+
+const cx = classNames.bind(styles);
+
+type MobileNavItemProps = {
+  title: string;
+  icon?: StaticImageData;
+  isDropdownActive: boolean;
+  color: string;
+  onMenuItemClick: (title: string) => void;
+};
+
+export const MobileNavItem = ({
+  onMenuItemClick,
+  icon,
+  isDropdownActive,
+  title,
+  color,
+}: MobileNavItemProps) => {
+  return (
+    <button onClick={() => onMenuItemClick(title)} className={cx('category-link', color)}>
+      <div className={cx('category-title')}>
+        {icon && (
+          <Image
+            src={icon}
+            alt="Donate-icon"
+            width={20}
+            height={18}
+            aria-hidden="true"
+          />
```

Comment:
I understand that we have only one icon now, but still it seems wrong to set in alt donate-icon - if the icon comes in props, maybe then together with the icon to pass the alt? And let's write “little heart” in this icon alt.